.gitignore

```diff
@@ -41,31 +28,6 @@ bld/
 # Visual Studio 2017 auto generated files
 Generated\ Files/
 
-# MSTest test Results
-[Tt]est[Rr]esult*/
-[Bb]uild[Ll]og.*
-
-# NUnit
-*.VisualState.xml
-TestResult.xml
-nunit-*.xml
-
-# Build Results of an ATL Project
-[Dd]ebugPS/
-[Rr]eleasePS/
-dlldata.c
-
-# Benchmark Results
-BenchmarkDotNet.Artifacts/
-
-# .NET Core
-project.lock.json
-project.fragment.lock.json
-artifacts/
-
-# StyleCop
-StyleCopReport.xml
-
```

Comment:
@Sotatek-HauTran3 I think you have deleted Windows platform related files. Is it okay when running in windows?

Reply:
It's ok

Reply:
It is also visual studio related [contents](https://github.com/klaytn/web3klaytn/pull/362/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L69)